awx/main/management/commands/provision_instance.py

```diff
@@ -35,7 +35,7 @@ def _register_hostname(self, hostname, node_type, uuid):
 
             from awx.main.management.commands.register_queue import RegisterQueue
 
-            (changed, instance) = Instance.objects.register(ip_address=os.environ.get('MY_POD_IP'), node_type='control', uuid=settings.SYSTEM_UUID)
+            (changed, instance) = Instance.objects.register(ip_address=os.environ.get('MY_POD_IP'), node_type='control', node_uuid=settings.SYSTEM_UUID)
```

Comment:
:eyes: 
not following why kwarg is called `node_uuid` if this is how field is declared on Instance model: 
https://github.com/ansible/awx/blob/036f85cd80430a894ea0ccb3212184fe1b04ac07/awx/main/models/ha.py#L86